build/build_cpp.sh

```diff
@@ -7,15 +7,28 @@ BASEDIR=$(dirname "$0")
 #Go to project directory
 cd $BASEDIR/../
 
-# Create build folder
-rm -rf ../build_llama_cpp
-mkdir -p ../build_llama_cpp
-cd ../build_llama_cpp
-
 if [ "$1" == "" ]; then
+	# Create build folder
+	rm -rf ../build_llama_cpp
+	mkdir -p ../build_llama_cpp
+	cd ../build_llama_cpp
+	
 	# Build library
 	cmake ../Project-Llama/llama_cpp
+elif [ "$1" == "Xcode" ]; then
+	# Create Xcode project folder
+	rm -rf ../llama_xcode
+	mkdir -p ../llama_xcode
+	cd ../llama_xcode
```

Comment:
Can also save some lines of codes as Xcode is a side building target, except from that, every building things should be in build_llama_cpp.